include/oneapi/dpl/internal/dynamic_selection.h

```diff
@@ -0,0 +1,205 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include <memory>
+#include <utility>
+#include <iostream>
+#include <sstream>
+
+#ifndef _ONEDPL_DYNAMIC_SELECTION_DEFS_H
+#define _ONEDPL_DYNAMIC_SELECTION_DEFS_H
+#pragma once
+
+
+namespace oneapi {
+namespace dpl {
+namespace experimental {
+//ds_properties
+
+  namespace property {
+    struct universe_t {
+      static constexpr bool is_property_v = true;
+      static constexpr bool can_report_v = false;
+    };
+    inline constexpr universe_t universe;
+
+    struct universe_size_t {
+      static constexpr bool is_property_v = true;
+      static constexpr bool can_report_v = false;
+    };
+    inline constexpr universe_size_t universe_size;
+
+    struct dynamic_load_t {
+      static constexpr bool is_property_v = true;
+      static constexpr bool can_report_v = false;
+    };
+    inline constexpr dynamic_load_t dynamic_load;
+
+    struct is_device_available_t {
+      static constexpr bool is_property_v = true;
+      static constexpr bool can_report_v = false;
+    };
+    inline constexpr is_device_available_t is_device_available;
+
+    struct task_execution_time_t {
+      static constexpr bool is_property_v = true;
+      static constexpr bool can_report_v = true;
+    };
+    inline constexpr task_execution_time_t task_execution_time;
+
+    struct task_submission_t {
+      static constexpr bool is_property_v = true;
+      static constexpr bool can_report_v = true;
+    };
+    inline constexpr task_submission_t task_submission;
```

Comment:
See if this is necessary for 1st release. May only be needed for dynamic load??  If it can be deferred, let's defer it.

Reply:
This has been removed